bakeware/lib/bakeware/assembler.ex

```diff
@@ -114,4 +117,22 @@ defmodule Bakeware.Assembler do
     File.chmod!(assembler.output, 0o755)
     assembler
   end
+
+  defp set_compression(assembler) do
+    compress? =
+      case :os.cmd('which zstd') do
```

Comment:
You could do `System.find_executable` here

Reply:
TIL